src/modules/calendarEventMethods.ts

```diff
@@ -94,9 +94,12 @@ async function updateEventSeries(params: UpdateCalendarEvent, user: User): Promi
         );
     }
 
+    const event = await calendarMethods.getEventById(eventId);
```

Comment:
you can use update to increment sequence and return a new value every time you need to

Reply:
Here and in splitEventSeries event updates in transactions and uses rule from `const event = await calendarMethods.getEventById(eventId)`, but I replaced getEventById with update where it possible